.htmlvalidate.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
     // Allow for conditional comments (used in header for fallback png)
     'no-conditional-comment': 'off',
 
+    // Allow implicit type="button" without type attribute
+    'no-implicit-button-type': 'off',
```

Comment:
This is still needed because we haven't shipped the changes in https://github.com/alphagov/govuk-frontend/pull/4113 yet, right?

I think if you don't have that context it's 'surprising' to see this included, given the other changes in this PR mean that we otherwise shouldn't need to disable the rule.

Suggest we either split this out, maybe into #3144, or at least add a note to the commit to explain why it's needed.

Reply:
Good thinking. I've removed it from this PR as the rule doesn't exist yet until https://github.com/alphagov/govuk-design-system/pull/3144